WebSurveyApp/WebSurveyApp/Controllers/QuestionController.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace WebSurveyApp.Controllers
+{
+    public class QuestionController : Controller
+    {
+        private readonly SurveyDbContext _context;
+
+        public QuestionController(SurveyDbContext context)
+        {
+            _context = context;
+        }
+
+        [HttpGet]
+        [Authorize]
+        public IActionResult Create([FromRoute] int surveyId)
+        {
+            var questionModel = new Question() { SurveyId = surveyId };
+
+            return View(questionModel);
+        }
+
+        [HttpPost]
+        [Authorize]
+        public async Task<IActionResult> Create([FromForm] Question questionModel)
+        {
+            if (ModelState.IsValid)
+            {
+                await _context.Questions.AddAsync(questionModel);
+                await _context.SaveChangesAsync();
+
+                int questionId = questionModel.Id;
+
+                return RedirectToAction($"Create", "Option", new { questionModel.SurveyId, questionId });
+            }
+
+            return View(questionModel);
+        }
+
+        [HttpGet]
+        [Authorize]
```

Comment:
You can move [Authorize] attribute to controller level. Or decorate all actions with it in Startup. Then you can use [AllowAnonymous] to bypass authorization where needed.